proposals/devcontainer-features.md

```diff
@@ -6,7 +6,25 @@ From a practical point of view, features are folders that contain units of code
 
 Features can be defined by a `devcontainer-feature.json` file in the root folder of the feature. The file is optional for backwards compatibility but it is required for any new features being authored.
 
-Features are to be executed in sequence as defined in `devcontainer.json`.
+By default, features are installed in an order selected by the implementing tool. 
+
+If any of the following properties in the feature's `devcontainer-feature.json, or the user's `devcontainer.json` are provided, the ordered indicated by the propert(ies) are respected.
+
+- `runsAfter` property defined as part of `devcontainer-feature.json`.
+- id.
+
```

Comment:
Should this bulleted list include `overrideFeatureInstallOrder`? Does it need to mention `id`?